CheckerPrinter/src/checkerprinter/InMemory.java

```diff
@@ -92,11 +92,35 @@ protected boolean initCheckerArgs(JsonObject optionsObject) {
             this.exceptionMsg = "Error: Cannot find indicated checker.";
             return false;
         }
-        this.checkerOptionsList = Arrays.asList( "-Xbootclasspath/p:" +
-                this.CHECKER_FRAMEWORK + "/checker/dist/jdk8.jar",
-                "-processor",
-                checker);
+        this.checkerOptionsList = new ArrayList<String>();
+
+        this.checkerOptionsList.add("-Xbootclasspath/p:" + this.CHECKER_FRAMEWORK + "/checker/dist/jdk8.jar");
+        this.checkerOptionsList.add("-processor");
+        this.checkerOptionsList.add(checker);
+
         if (optionsObject.getBoolean("has_cfg")) {
+
+            //ugly, how can this be improved?
+            if(! setupDir(new File("../CFG"))){              
```

Comment:
As we discussed before, at current stage you can ignore the multi-user call problem and focus on implementing the main functions. So as a temporary workaround, we can just manually create a `CFG` directory in the server and here just assume there should have a `CFG` directory in the expected location.